### PR TITLE
Fixed bug for Sydney region where zone and region values are the same

### PR DESCRIPTION
**Pull request by taimai**

### CODE REVIEW

File: src/atlantis/manager/helper/helper.go [hunk 1]
```diff
@@ -192,7 +193,15 @@ func RegionAndZone(zone string) string {
...
 func ZoneMinusRegion(zone string) string {
-	return strings.Replace(zone, Region, "", 1)
+	zmr := strings.Replace(zone, Region, "", 1)
+	if zmr == "" {
+		results := strings.Split(zone, "-")
+		if len(results) == 2 {
+			zmr = results[1]
+		}
+	}
+	log.Printf("ZoneMinusRegion: zone=%s Region=%s zmr='%s'", zone, Region, zmr)
```
Comment (edanaher) [inline src/atlantis/manager/helper/helper.go]: I'm not sure we want to be logging this regularly; I would expect that this function is called quite a bit, and this doesn't provide insight into how the services is working, aside from demonstrating that this function is correct.

Comment (taimai) [inline src/atlantis/manager/helper/helper.go]: It's only called when we need to register a router so I don't think you'll see the log that often.

**Comment by blackgold** [comment]
LGTM

File: src/atlantis/manager/helper/helper.go (hunk 1 above)
Comment (edanaher) [inline src/atlantis/manager/helper/helper.go]: Fair enough.

**Pull request merged**